src/material/sort/sort-header-intl.ts

```diff
@@ -20,16 +20,8 @@ export class MatSortHeaderIntl {
    * components if the labels have changed after initialization.
    */
   readonly changes: Subject<void> = new Subject<void>();
-
-  /**
-   * ARIA label for the sorting button.
-   * @deprecated Not used anymore. To be removed.
-   * @breaking-change 8.0.0
-   */
-  sortButtonLabel = (id: string) => {
-    return `Change sorting for ${id}`;
-  }
 }
+
```

Comment:
@andrewseguin should we mark `MatSortIntl` as deprecated? It isn't being used for anything anymore.

Reply:
Yeah sounds good to me